kinyarwanda/1.1994 Genocide against tutsi in Rwanda/1994-genocide-in-Rwanda

```diff
@@ -0,0 +1 @@
+In 1994, Rwanda faced a terrible genocide with a loss of more than 1 millions of lives of tustsi.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.